data/transactions/logic/eval.go

```diff
@@ -2014,19 +2024,46 @@ func opB(cx *EvalContext) error {
 	return nil
 }
 
+const protoByte = 0xf0
```

Comment:
nit: sorry for coming back to this line, but I noticed in #4442, you introduced
```golang
OpsByName[ops.Version]["intc_0"].Opcode
```
to tell the bytes of the opcode, I wonder if it is necessary to apply here, or I am imaging something wrong?

Reply:
Yes, it would be good to use that here too.

Reply:
Well, actually, maybe not.  That involves a table lookup, and it's dependent on Version, so it would slow down the code that needs this, and here we're in evaluation code, not assembly, so I don't want it slow.  Instead, I'll add a unit test to ensure this constant is correct, in case we ever consider moving the opcode.

Reply:
ah, fair enough. we are at a point that bytes are still consistent with version, so indeed no need for table lookup.

Reply:
I forgot that I had done this, which seems good enough for documenting that protoByte has to be the opcode for "proto".
```
// protoByte is a named constant because opCallsub needs to know it.
{protoByte, "proto", opProto, proto(":"), fpVersion, immediates("a", "r")},
```